### ISSUE TITLE
plugin compile instructions incorrect?

### ISSUE THREAD

**Issue by d834256** [ddelnano/packer-plugin-xenserver#109]
Plugin compile instructions in the README -

```
$ go build -o packer-plugin-xenserver

# Add the builder to the location packer expects it to be installed in
$ mkdir -p ~/.packer.d/plugins/
$ cp builder-xenserver-iso  ~/.packer.d/plugins/packer-builder-xenserver-iso
```

Should that last line be as follows?

...
$ cp packer-plugin-xenserver  ~/.packer.d/plugins/packer-builder-xenserver-iso
...

**Comment by ddelnano** [comment]
The name of the file and location of it has changed between packer versions iirc. Can you explain what problem you are facing in more detail and what commands you tried?

**Comment by d834256** [comment]
The go build command builds a binary called packer-plugin-xenserver, the first argument to the cp command is builder-xenserver-iso which doesn't exist.

**Comment by ddelnano** [comment]
Pull requests are greatly appreciated, so a fix like that is welcome :). What I'm not sure of is if the `cp` command's destination is correct for the latest packer versions.

The packer plugin directory (`~/.packer`) and the plugin naming convention has changed over the years, so it would be good to fix the inconsistency you caught in addition to validating those instructions work end to end.

**Comment by d834256** [comment]
Roger that! I'm working through various things with the plugin at the moment, particularly with Windows, so I'll try and come back to this.

**Comment by Shackelford-Arden** [comment]
@d834256 You didn't note which issues you were having on Windows, but I was able to build it on Windows using the steps I included in #111 .

Those steps are:

```powershell
go build -o packer-plugin-xenserver

mkdir "%APPDATA%\packer.d\plugins"
cp packer-plugin-xenserver  "%APPDATA%\packer.d\plugins"
```

I've also adjusted the Linux/MacOS script to be consistent on the naming as well.

**Comment by ddelnano** [comment]
This is complete with #111 merged. Thanks again @Shackelford-Arden!
